src/Couchbase.Lite.Shared/Sync/WebSocketWrapper.cs

```diff
@@ -564,6 +573,7 @@ private async void PerformWrite()
                         if (!_closed) {
                             unsafe {
                                 Native.c4socket_completedWrite(_socket, (ulong) nextData.Length);
+                                WriteLog.To.Sync.I(Tag, "c4Socket completed Write.");
```

Comment:
This one is going to get noisy so I think it should be left out, or moved to verbose.